00 Intro/01 Import/README.md

```diff
@@ -1,149 +1,174 @@
 # 01 Import
 
-In this sample we are going to start working with es6 modules (import).
+In this sample we are going to start working with ES6 modules (import).
 
-We will start from sample _00 Intro/00 Boilerplate_ and add a new JavaScript that will
+We will start from sample _00 Intro/00 Boilerplate_ and add a new JavaScript service that will
 hold a simple algorithm to calculate the average of an score array.
 
 We will use this JavaScript array into the main students.js file by importing
 it.
 
 Summary steps:
- - Add a new file averageService.js
- - Add an array on students.js
- - Import averageService into students.js
- - Use the averageService inside the students.js code.
- - Transpile and test on index.html
+ - Add a new file `averageService.js`
+ - Add an array on `students.js`
+ - Import averageService into `students.js`
+ - Use the averageService features inside the `students.js` code.
+ - Transpile and test on `index.html`
 
 
 # Steps to build it
 
 ## Prerequisites
 
-Prerequisites, you will need to have nodejs installed in your computer. If you want to follow this step guides you will need to take as starting point sample _00 BoilerPlate_.
+You will need to have Node.js installed in your computer. In order to follow this step guides you will also need to take sample _00 BoilerPlate_ as a starting point.
 
-## steps
+## Steps
 
-- `npm install` to install previous sample packages:
+- `npm install` to install previous sample dependencies:
 
-```
-npm install
-```
+  ```
+  npm install
+  ```
 
-- Let's add a new file called averageService.js, this file will contain a function that will calculate the average value of a given array, this function will be exported (make it visible to other modules that need to consume them).
+- Let's add a new file called `averageService.js`. This file will contain a function that will calculate the average value of a given array, this function will be exported (make it visible to other modules that need to consume them).
 
 ### ./averageService.js
```

Comment:
Hi @fjcalzado, I mean for example:

- Add the following content to `averageService.js`:

### ./averageService.js
```javascript
export function ....
....
```